x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/inventory_metric_threshold_executor.ts

```diff
@@ -255,6 +263,8 @@ export const createInventoryMetricThresholdExecutor = (libs: InfraBackendLibs) =
       const alertUuid = getAlertUuid(recoveredAlertId);
       const alertHits = alertUuid ? await getAlertByAlertUuid(alertUuid) : undefined;
       const additionalContext = getContextForRecoveredAlerts(alertHits);
+      const getOriginalAlertState = createGetOriginalAlertState(actionGroupToAlertState);
```

Comment:
It took me a while to understand that this actually is about translating the state value. Maybe the function names and lookup tables could represent that?